sdk/selfhelp/Azure.ResourceManager.SelfHelp/src/Generated/Models/ContinueRequestBody.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.SelfHelp.Models
+{
+    /// <summary> Troubleshooter ContinueRequest body. </summary>
+    public partial class ContinueRequestBody
+    {
+        /// <summary> Initializes a new instance of ContinueRequestBody. </summary>
+        public ContinueRequestBody()
+        {
+            Responses = new ChangeTrackingList<TroubleshooterResponse>();
+        }
+
+        /// <summary> Unique id of the result. </summary>
+        public string StepId { get; set; }
```

Comment:
What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://github.com/Azure/azure-sdk-for-net/blob/bd953fc7a97bf617c892ce6314b50126b4021a4f/sdk/maps/Azure.ResourceManager.Maps/src/autorest.md?plain=1#L25).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://github.com/Azure/azure-sdk-for-net/blob/bd953fc7a97bf617c892ce6314b50126b4021a4f/sdk/search/Azure.Search.Documents/src/autorest.md?plain=1#L106-L115).